### ISSUE TITLE
Permission to access threads I participated as a retired member

### ISSUE THREAD

**Issue by madduck** [loomio/loomio#8241]
I was a member of a Committee last year, and am no longer. Thus, I've been removed from the Loomio group. I regularly get invited to individual threads, which is a great feature of Loomio. However, I am unable to access threads that I participated in, even the ones I authored in that group, when arguably, I should be considered to have been invited to these threads by virtue of having participated.

Obviously there exist cases where access needs to be revoked entirely, but maybe a new state needs to be introduced such that group members can be "retired", allowing them access to the existing material, just not anything new.

Thanks for your consideration

**Comment by robguthrie** [comment]
Oh ok. When you're removed from a group, you're removed from all the
threads you were invited to while a member of the group. That's what most
users expect to happen.
I could add an option about this but it would be really hard to make user
friendly.

Were you added to this thread before you were removed from the group?

**Comment by robguthrie** [comment]
Ok I'll consider adding the "remove from group but keep access to threads
already invited to".

Congrats on once again being at the leading edge of complex use cases :)

**Comment by robguthrie** [comment]
After a minute's thought, I don't think that is the right way. I reckon it will be more secure by default to just remove access completely when you remove a member, but have a "request access" button on the permission denied (not working) screen.

**Comment by madduck** [comment]
I agree on removing members, and that this should remove their rights as it currently does. My thinking is that it would be good to be able to retire users such that from this point onwards, they cannot see new content, only everything up until the point of their retirement.

And sorry for being "on the leading edge of complex cases" — I am just using Loomio and letting you know when I run up against issues.

**Comment by robguthrie** [comment]
Oh, that was not a jab, I'm grateful for you being so helpful in working through these edge-cases. 

I understand what you're saying about this retired user use-case. I'm not sure about it, because existing threads with new comments could be a problem, and it's not super clear if someone is in or out. I'm just not sure it's worth the complexity vs just removing someone then adding them back to specific threads they need access to.
